babel.config.js

```diff
@@ -36,7 +36,12 @@ module.exports = {
       },
     ],
     '@babel/plugin-transform-object-assign',
-    '@babel/plugin-transform-runtime',
+    [
+      '@babel/plugin-transform-runtime',
+      {
+        version: '^7.0.0',
```

Comment:
This is the default as well as ["mostly uneffective"](https://github.com/babel/website/pull/2134#pullrequestreview-323343193). It is only added for clarity.